lib/Listener/UserDeletedListener.php

```diff
@@ -52,13 +52,18 @@ public function handle(Event $event): void {
 		$user = $event->getUser();
 
 		$rooms = $this->manager->getRoomsForUser($user->getUID());
-
 		foreach ($rooms as $room) {
 			if ($this->participantService->getNumberOfUsers($room) === 1) {
 				$room->deleteRoom();
 			} else {
 				$this->participantService->removeUser($room, $user, Room::PARTICIPANT_REMOVED);
 			}
 		}
+
+		$leftRooms = $this->manager->getLeftOneToOneRoomsForUser($user->getUID());
+		foreach ($leftRooms as $room) {
+			$room->setType(Room::GROUP_CALL, true);
```

Comment:
this means other users could be invited to that conversation and then they could read past messages, feels like an unwanted side effect.
if we had a way to mark it as archived, like read-only mode but would need an external system user moderator then to prevent anyone to remove there read-only flag

Reply:
No one cares? Don't want to make it overly complicated so that is good enough. Before you could do screenshares and share them already `¯\_(ツ)_/¯`

I'm fine with marking it archived by default as well. We could also "demote" the other user to be a normal user and they can then leave the conversation only. But simplicity is the key here I think.

Reply:
I've tested it and seems fine then.

A good thing is that now that it's a group conversation, the remaining user is able to delete it also.